.github/workflows/test.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           sudo apt install -y libegl1-mesa libgl1-mesa-dri libxcb-xfixes0-dev mesa-vulkan-drivers
 
       - name: run checks & tests
-        run: ./run-checks.sh std
+        run: ./run-checks.sh std ci
```

Comment:
We should not have a flag that changes behavior for CI and local execution. The interface should be made simple with one target. This will be treated like a `make [target]` command.